src/components/hooks/useAccordion/useAccordion.ts

```diff
@@ -1,14 +1,48 @@
-import { useRef, useState, Dispatch, SetStateAction } from 'react';
+import { useRef, useState, Dispatch, SetStateAction, KeyboardEvent, RefObject } from 'react';
 import { isArrowDown, isArrowUp } from '../../../helpers/keyboard-keys';
 import { mod } from '../../../helpers/utils';
 
+interface IAccordionHeaderProps {
+  'aria-controls': string;
+  'aria-disabled': boolean;
+  'aria-expanded': boolean;
+  id: string;
```

Comment:
If we end-up setting `id` on the HTML elements rendered by `<Accordion.Header />` and `<Accordion.Section />` there's a chance that we end with two duplicate `id`s on the document ?
```jsx
<Accordion aria-label="test-accordion">
  <Accordion.Header id="one" />
  <Accordion.Section id="one" />
</Accordion>;
```
I believe that's bad as `id` should be unique? Maybe we could use `data-accordion-id` instead?

Reply:
this `id` doesn't translate directly to the `id` attribute in case. The above example would result in:
```
<Accordion.Header id="one" />
<Accordion.Section id="one-section" />
```